dev/blaze/dev.clj

```diff
@@ -34,11 +34,13 @@
 ;; Init Development
 (comment
   (init)
+  system
```

Comment:
do you really need the system here? I mean it's available above.

Reply:
You mean below?
That's why there are two comment blocks.

In this way, one can call the system var to verify the system is actually running in both cases without having to switch comment blocks.

Reply:
Yes on line 22. I don't need to eval the system on regular basis. Why do you need it to tell that the system is running? If I doesn't;t run, there will be an exception during startup. Otherwise it runs.